test/stack_test.c

```diff
@@ -45,6 +45,8 @@ DEFINE_SPECIAL_STACK_OF_CONST(uchar, unsigned char)
 DEFINE_STACK_OF(SS)
 DEFINE_STACK_OF_CONST(SU)
 
+IMPLEMENT_COMPARISONS(char, char, "%c")
```

Comment:
I figured having some global implementations for common case comparisons would be useful.
However, there is no reason to not force each test program using these to declare exactly those they use themselves.  I'm kind of ambivalent here -- I'm happy to go either way and is seems the consensus is pushing for pertest instantiation.